README.md

```diff
@@ -107,6 +107,22 @@ If not one of the well-described tests, the Fake Provider exhibits the following
 - If the CPR number ends in `000`, `001` or `002` it returns NotFound
 - The Fake provider returns random data, using the CPR number as the seed. This ensures recurring calls return the same response
 
+## CPR Subscription
+
+Whenever there is a change in information about a Citizen the CPR provider will raise events.
+ 
+Example -
+ [Datafordeler (Events)](https://confluence.datafordeler.dk/pages/viewpage.action?pageId=17137834#H%C3%A6ndelserp%C3%A5Datafordeleren-Brugsscenarier)
+ 
+- Subscribe To The Events By CPR number or by PersonId
```

Comment:
Subscribe to the events by CPR Number or by Person Id

Reply:
changed as suggested